Receipts/kubernetes/dnsutils/dnsutils.yml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
```

Comment:
i think thats not needed as well, think you did that to see how the dns resolve but its not part of the project =D

Reply:
You mean the whole dnsutils.yml file?